test/studies/adventOfCode/2022/day07/bradc/day07-par.future

```diff
@@ -0,0 +1 @@
+feature request: maps should support parallel iterators (sigh...)
```

Comment:
I've been circling the idea recently that we need to make sure the `parSafe` handling in maps gets split either into the distributed map or into its own parallel map implementation before 2.0, as part of stabilizing the map type.  Stuff like this makes it more obvious that we haven't tested the parallel corner cases well